docs/environment-variables.adoc

```diff
@@ -66,8 +66,15 @@ module java.base does not "opens java.lang" to unnamed module [...]
 ----
 
 The best way to prevent these warnings/errors, is to change the code under test, so this extension is no longer needed.
-The next best thing is to allow access to that specific package with `--add-opens=java.base/java.util=ALL-UNNAMED` (if you place JUnit Pioneer on the class path) or `--add-opens=java.base/java.util=org.junitpioneer` (if you place it on the module path).
+The next best thing is to allow access to that specific package:
 
+[source]
+----
+--add-opens java.base/java.util=$TARGET_MODULE
```

Comment:
So `java.util` also needs to be opened for prod (not just for test)?

Reply:
Without `java.util`:

```text
EnvironmentVariable extension > initializationError FAILED
    org.junit.jupiter.api.extension.ExtensionConfigurationException: Cannot access Java runtime internals to modify environment variables. Have a look at the documentation for possible solutions: https://junit-pioneer.org/docs/environment-variables/#warnings-for-reflective-access
        at app/org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableUtils.getFieldValue(EnvironmentVariableUtils.java:104)
        at app/org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableUtils.setInSystemEnvClass(EnvironmentVariableUtils.java:91)
        at app/org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableUtils.trySystemEnvClass(EnvironmentVariableUtils.java:60)
        at app/org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableUtils.modifyEnvironmentVariables(EnvironmentVariableUtils.java:53)
        at app/org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableUtils.set(EnvironmentVariableUtils.java:36)
        at app/org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableExtensionTests.globalSetUp(EnvironmentVariableExtensionTests.java:39)

        Caused by:
        java.lang.reflect.InaccessibleObjectException: Unable to make field private final java.util.Map java.util.Collections$UnmodifiableMap.m accessible: module java.base does not "opens java.util" to module org.junitpioneer
            at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
            at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
            at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
            at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
            at org.junitpioneer/org.junitpioneer.jupiter.EnvironmentVariableUtils.getFieldValue(EnvironmentVariableUtils.java:99)
```

So it is because of the `java.util.Map` we try to make accessible. Will add a comment as you suggested above.